File: JoelMcBethWebsite/Data/Models/Book.cs
namespace JoelMcBethWebsite.Data.Models
{
    using System;
    using System.Collections.Generic;
    using System.Collections.ObjectModel;
    using System.Linq;
    using System.Threading.Tasks;

    public class Book
    {
        public Book()
        {
            this.Authors = new Collection<Author>();
            this.Reviews = new Collection<BookReview>();
        }

        public int Id { get; set; }

        public string Isbn13 { get; set; }

        public string Title { get; set; }

        public string Edition { get; set; }

        public int? Pages { get; set; }

        public ICollection<Author> Authors { get; set; }

        public int? Order { get; set; }

        public ICollection<BookReview> Reviews { get; set; }
    }
}

File: JoelMcBethWebsite/ClientApp/src/app/books/book.ts
import { Author } from "./author";
import { BookReview } from "./book-review";

export interface Book {
    id: number;    
    isbn13: string;
    title: string;
    edition: string;
    pages: number;
    order: number;
    authors: Author[];
    reviews: BookReview[];
}
